checkstyle/checkstyle.xml

```diff
@@ -64,6 +64,9 @@
     <module name="ParameterName"/>
     <module name="StaticVariableName"/>
     <module name="TypeName"/>
+    <module name="AvoidStarImport">
+      <property name="allowClassImports" value="false"/>
```

Comment:
Yes, a single line &lt;module name="AvoidStarImport" /&gt; would suffice. I just tested this on my local and it works. I will re-issue pull request with this update.

Thanks again for your feedback!